Qfront/src/Pages/Shop/Shop.jsx

```diff
@@ -78,11 +49,11 @@ const Shop = () => {
 
   // UseEffect to load when component mounts. Fetches all categories and fetches all products
   useEffect(() => {
-    init();
+    dispatch(getAllCategories())
     loadFilteredResults(limit, skip, myFilters.filters);
```

Comment:
Did you test this? Passing 3 parameters to the action? The `options` parameter is only the first one